vyxal/elements.py

```diff
@@ -2835,8 +2842,10 @@ def repeat(lhs, rhs, ctx):
         @lazylist
         def gen():
             prev = value
+            val = value
             while True:
-                val = safe_apply(function, value, ctx=ctx)
+                val = safe_apply(function, val, ctx=ctx)
+                print(val)
```

Comment:
Please avoid leaving `print`s lying around

Reply:
Good catch - I missed that bit of debugging last night.